HWs/HW4/ParseTree/ParseTree/ParseTreeUtils/Operation.cs

```diff
@@ -0,0 +1,32 @@
+namespace ParseTree;
+
+/// <summary>
+/// Parse tree element - operation.
+/// </summary>
+internal class Operation : IParseTreeElement
+{
+    /// <summary>
+    /// Left subtree.
+    /// </summary>
+    public IParseTreeElement? LeftChild { get; set; }
```

Comment:
```suggestion
    public IParseTreeElement LeftChild { get; set; }
```
Зачем тут `Nullable`? Разве возможен случай, когда у операции не хватает одного или двух операндов?